src/docfx/build/git/Git.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class Git
+    {
+        public static void Process(PageModel model, Document document, GitRepoInfoProvider repo)
+        {
+            Debug.Assert(document != null);
+            Debug.Assert(repo != null);
+            if (!repo.ProfileInitialized)
+                return;
+
+            // TODO: support specifed authorName and updatedAt
+            GitUserProfile authorInfo = null;
```

Comment:
Isn't author calculated from `author` yaml header?

Reply:
the entire scenario is:
1. if set in Yaml header of docfx config, use that value;
2. otherwise calculate from Git history.

---
In reply to: [197010049](https://github.com/dotnet/docfx/pull/2912#discussion_r197010049) [](ancestors = 197010049)

Reply:
Yes. I have a TODO for this.

---
In reply to: [197010049](https://github.com/dotnet/docfx/pull/2912#discussion_r197010049) [](ancestors = 197010049)